src/pages/footer/Footer.jsx

```diff
@@ -0,0 +1,64 @@
+import { LinkButton as Link } from "src/components/Button/Button";
+import { Wrapper } from "./Footer.styled";
+
+export default function Footer() {
+  return (
+    <Wrapper as="footer" display="flex" justifyContent="space-between">
+      <div className="footer-container">
+        <ul className="footer-links">
+          <li>
+            <Link>EXPLORE OUR</Link>
```

Comment:
Here, you can store all these links in an array and then can traverse on that.

Like,
```
["EXPLORE OUR", "Labal", "Events"].map((link) => <Link>{link}</Link>)
```

I think these links are not active, as you have not used `to` attribute on them, so whenever you will work you can do that.